build-ps/percona-server-5.7_builder.sh

```diff
@@ -0,0 +1,825 @@
+    if [ $INSTALL = 0 ]
+    then
+        echo "Dependencies will not be installed"
+        return;
+    fi
+    if [ ! $( id -u ) -eq 0 ]
+    then
+        echo "It is not possible to instal dependencies. Please run as root"
+        exit 1
+    fi
+    CURPLACE=$(pwd)
+    
+    if [ "x$OS" = "xrpm" ]; then
+        RHEL=$(rpm --eval %rhel)
+        ARCH=$(echo $(uname -m) | sed -e 's:i686:i386:g')
+        add_percona_yum_repo
+        yum -y install http://www.percona.com/downloads/percona-release/redhat/0.1-4/percona-release-0.1-4.noarch.rpm || true
+        yum -y install epel-release
+        yum -y install git numactl-devel rpm-build gcc-c++ gperf ncurses-devel perl readline-devel openssl-devel jemalloc 
+        yum -y install time zlib-devel libaio-devel bison cmake pam-devel libeatmydata jemalloc-devel
+        yum -y install perl-Time-HiRes libcurl-devel openldap-devel unzip wget libcurl-devel 
+        yum -y install perl-Env perl-Data-Dumper perl-JSON MySQL-python perl-Digest perl-Digest-MD5 perl-Digest-Perl-MD5 || true
+        if [ ${RHEL} -lt 7 -a $(uname -m) = x86_64 ]; then
+            yum -y install percona-devtoolset-gcc percona-devtoolset-gcc-c++ percona-devtoolset-binutils 
+        fi
+        if [ "x$RHEL" = "x6" ]; then
+            yum -y install Percona-Server-shared-56  
+        fi
+    else
+        apt-get -y install dirmngr || true
+        add_percona_apt_repo
+      	apt-get update
+        apt-get -y install dirmngr || true
+        apt-get -y install lsb-release wget
+        export DEBIAN_FRONTEND="noninteractive"
+        export DIST="$(lsb_release -sc)"
+
+	    until sudo apt-get update; do
+    	    sleep 1
+            echo "waiting"
+        done
+        apt-get -y purge eatmydata || true
+        echo "deb http://jenkins.percona.com/apt-repo/ ${DIST} main" > percona-dev.list
+        mv -f percona-dev.list /etc/apt/sources.list.d/
+        wget -q -O - http://jenkins.percona.com/apt-repo/8507EFA5.pub | sudo apt-key add -
+        wget -q -O - http://jenkins.percona.com/apt-repo/CD2EFD2A.pub | sudo apt-key add -
+
+        apt-get update
+        apt-get -y install psmisc
+        apt-get -y install libsasl2-modules:amd64 || apt-get -y install libsasl2-modules
+        apt-get -y install dh-systemd || true
+        apt-get -y install curl bison cmake perl libssl-dev gcc g++ libaio-dev libldap2-dev libwrap0-dev gdb unzip gawk
+        apt-get -y install lsb-release libmecab-dev libncurses5-dev libreadline-dev libpam-dev zlib1g-dev libcurl4-openssl-dev
+        apt-get -y install libldap2-dev libnuma-dev libjemalloc-dev libeatmydata libc6-dbg valgrind libjson-perl python-mysqldb libsasl2-dev
+
+        apt-get -y install libmecab2 mecab mecab-ipadic
+        apt-get -y install build-essential devscripts
+        apt-get -y install cmake autotools-dev autoconf automake build-essential devscripts debconf debhelper fakeroot 
+
+
+    fi
+    if [ ! -d /usr/local/percona-subunit2junitxml ]; then
```

Comment:
not sure if we need subunit. is it for Jenkins param/trunk builds?

Reply:
we use awk for long time already

Reply:
done